libs/features/webapp-prompt/webapp-prompt.css

```diff
@@ -188,3 +188,119 @@
     }
   }
 }
+
+/* DISMISSAL TOOLTIP */
+
+[data-pep-dismissal-tooltip]::after {
+  content: attr(data-pep-dismissal-tooltip);
+  display: inline-flex;
+  z-index: 3;
+  background-color: #3B63FB;
+  height: fit-content;
+  width: 8.875rem;
+  top: 125%;
+  left: -300%;
+  word-break: break-word;
+  border-radius: 7px;
+
+  padding-inline-start: 0.5625rem;
+  padding-inline-end: 0.5625rem;
+  padding-block-start: 0.25rem;
+  padding-block-end: 0.3125rem;
+
+  font-family: 'Adobe Clean';
+  font-size: 0.75rem;
+  line-height: 0.9375rem;
+  color: white;
+}
+
+[data-pep-dismissal-tooltip]::before {
+  content: '';
+  z-index: 2;
+  background-color: #3B63FB;
+  width: 0.44rem;
+  height: 0.44rem;
+  border-radius: 0.05rem;
+  left: calc(50% - 0.22rem);
+  top: 115%;
+
+  transform: rotate(45deg);
+}
+
+[data-pep-dismissal-tooltip]::before,
+[data-pep-dismissal-tooltip]::after {
+  background-color: #3B63FB;
+  position: absolute;
+  pointer-events: none;
+  transition: opacity 0.5s;
+}
+
+@media (min-width: 1520px) {
+  [data-pep-dismissal-tooltip]::after {
+    left: calc(50% - 5rem);
+  }
+}
+
+/* DISMISSAL ANIMATION */
+
+.coach-indicator {
+  --coach-indicator-ring-default-color: rgba(56,146,243);
+  --coach-indicator-ring-diameter: 1.25rem;
+	--coach-indicator-ring-border-size: 2px;
+	--coach-indicator-ring-inline-size: var(--coach-indicator-ring-diameter);
+	--coach-indicator-ring-block-size: var(--coach-indicator-ring-diameter);
+  --animation-duration: 3000ms;
+}
+
+@keyframes pulse {
+	0% {
+		transform: scale(0.8);
+		opacity: 0;
+	}
+
+	50% {
+		transform: scale(1.5);
+		opacity: 1;
+	}
+
+	100% {
+		transform: scale(2);
+		opacity: 0;
+	}
+}
+
+.coach-indicator .coach-indicator-ring {
+	display: block;
+	position: absolute;
+  top: 14%;
+  left: 13%;
+
+	border-style: solid;
+	border-width: var(--coach-indicator-ring-border-size);
+	border-color: var(--coach-indicator-ring-default-color);
+
+	inline-size: var(--coach-indicator-ring-inline-size);
+	block-size: var(--coach-indicator-ring-block-size);
+	animation: pulse var(--animation-duration) linear;
+  animation-fill-mode: both;
+
+  border-radius: 5px;
+}
+
+.coach-indicator .coach-indicator-ring:nth-child(1) {
+  animation-delay: calc(var(--animation-duration)*(0));
```

Comment:
Couldn't this be just `0`?

Reply:
I'd kept them like this so I could fiddle with them. I've made it into a variable.